apigw-lambda-cognito-sam-java/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.example</groupId>
+  <artifactId>apigw-lambda-cognito</artifactId>
+  <name>Cognito user management</name>
+  <version>1.0-SNAPSHOT</version>
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>3.0.0-M5</version>

```

Comment:
any reason not to use the latest version?

Reply:
Fixed.

Reply:
According to https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-surefire-plugin the latest version is `3.3.1`?

Reply:
Fixed the pom.xml file and removed dependency-reduced-pom.xml as that is not required to be checked in.